### ISSUE TITLE
FancyChickenModel crash on NeoForge 1.21

### ISSUE THREAD

**Issue by parmerinha99** [Slexom/earth2java#145]
I'm having this crash on neoforge 1.21 

 

java.lang.IllegalAccessError: class slexom.earthtojava.client.renderer.entity.model.FancyChickenModel tried to access private field net.minecraft.client.model.ChickenModel.head (slexom.earthtojava.client. renderer.entity.model.FancyChickenModel is in module earthtojavamobs@1.11.3 of loader 'TRANSFORMER' @54c425b1; net.minecraft.client.model.ChickenModel is in module minecraft@1.21 of loader 'TRANSFORMER' @54c425b1)

**Comment by Amateur233** [comment]
READ ITS MODPAGE, IT'S DEPERCATED. ONLY FOR FORGE 1.16.1

**Comment by Amateur233** [comment]
SERIOUSLY HOW ARE PEOPLE THIS BLIND

**Comment by Amateur233** [comment]
You're using https://www.curseforge.com/minecraft/mc-mods/earth2java-mobs
For the love of god, read the modpage's description.

**Comment by parmerinha99** [comment]
No, I'm using the earth2java-neoforge-1.11.4+1.21 file from this link:
https://www.curseforge.com/minecraft/mc-mods/earth2java/files/5526906

I'm just reporting what happened in my game, no need to get mad about it. Be a little more polite to those who are trying to help you improve your beautiful work.

**Comment by Slexom** [comment]
Cleaned the thread... anyway...
Looks like you are using 1.11.3 
From the error `earthtojavamobs@1.11.3 `
That version is archived and should not be visible, are you using a modpack? In case replace it with 1.11.4(is marked as beta) and let me know.

**Comment by xErrorAmelie** [comment]
I have the same issue with Earth2Java 1.11.4 Neoforge.
Here is the crash-report: https://mclo.gs/EdnAXAz
